Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -369,13 +369,33 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
                 var groupElement = group.render(messageView);
                 this._currentSessionOrGroup.append(groupElement);
                 this._currentSessionOrGroup = group;
-            } else
-                this._currentSessionOrGroup.addMessageView(messageView);
+            } else this._currentSessionOrGroup.addMessageView(messageView);
+            
+            if (WI.settings.experimentalGroupSourceMapErrors.value && WI.consoleManager.failedSourceMapConsoleMessages.has(messageView.message))
+                this._addConsoleMessageToConsoleSourceMapErrorGroup(messageView);
         }
 
         if (this.delegate && typeof this.delegate.didAppendConsoleMessageView === "function")
             this.delegate.didAppendConsoleMessageView(messageView);
     }
+    
+    _addConsoleMessageToConsoleSourceMapErrorGroup(messageView)
+    {
+            
+        if (this._firstSourceMapErrorMessageView === undefined) {
+            this._firstSourceMapErrorMessageView = messageView;
+            return;
+        }
+        
+        if (this._failedSourceMapsGroup === undefined) {
```

Comment:
ditto

Reply:
Addressed in e2c3874b5b35844fbcd028d68867b71662ae988e.